### DISCUSSION TITLE
tinymce not load

### DISCUSSION THREAD

**Discussion by Danieleeffe1** [OpenMage/magento-lts#3732] [Questions & Answers] [answered]
Good morning everyone.
 

I upgraded OM 20.1 to OM 20.3 by downloading the source OM 20.3 and overwriting the files of OM 20.1 . 

the server is Apache 2 with php 7.4.

I have a problem with TinyMCE

this in fact is not displayed

suggestions?

**Comment by fballiano** [comment]
do you see an error on the browser's console?

**Reply by Danieleeffe1** [reply]
Hi thank you for your response.

No in the Chome console no errors are highlighted

**Comment by addison74** [comment]
If I remember correctly, we recently removed TinyMCE from OpenMage. It is installed with Composer.

**Reply by fballiano** [reply]
no that PR was reverted because it didn't work

**Comment by pquerner** [comment]
Did you clear cache and force the browser to hard-reload the website?

**Comment by addison74** [comment]
I tried without success to reproduce this issue. I have attached proof that the plugin works as expected when I want to edit a page from the CMS section. 

**I used this test environment**

- OpenMage v20.0.3 (Windows 10 + WSL + Ubuntu + Docker + DDEV)
- Magento Sample Pack
- PHP 7.4

![tiny_editor](https://github.com/OpenMage/magento-lts/assets/8360474/f71852c8-50d7-441b-a5e9-22d4e16f7579)

**Comment by Danieleeffe1** [comment]
I don't know why it doesn't work for me, I'm reinstalling now.

I thank everyone for your availability

**Comment by Danieleeffe1** [comment, marked as answer]
I found out what is preventing TinyMCE from loading.

Years ago I purchased (and still use) a responsive backend theme to be able to use Magento OpenMage with mobile devices.

If I uninstall this theme tinyMCE loads normally.

**Reply by Danieleeffe1** [reply]
information added: the developer of this theme is no longer operational/available
